docs/windows.md

```diff
@@ -0,0 +1,187 @@
+```
+$cmd = "C:\k\kubelet.exe $global:KubeletArgs --cert-dir=$env:SYSTEMDRIVE\var\lib\kubelet\pki --config=/var/lib/kubelet/config.yaml --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --kubeconfig=/etc/kubernetes/kubelet.conf --hostname-override=$(hostname) --pod-infra-container-image=`"mcr.microsoft.com/k8s/core/pause:1.3.0`" --enable-debugging-handlers --cgroups-per-qos=false --enforce-node-allocatable=`"`" --network-plugin=cni --resolv-conf=`"`" --log-dir=/var/log/kubelet --logtostderr=false --image-pull-progress-deadline=20m --node-ip=$env:NODE_IP"
+```
+
+3.Prepare network adapter for kubernetes service.
+Kube-proxy need a network adapter to configure kubernetes service IPs and use the adapter for network proxy.
+```
+curl.exe -LO https://raw.githubusercontent.com/ruicao93/antrea/windows_install/hack/windows/Prepare-ServiceInterface.ps1
+.\Prepare-ServiceInterface.ps1
+```
+
+3.Run kubeadm to join the node.
+
+On windows worker node, use the command that was given to you when you ran `kubeadm init` on a control plane host to join the node to cluster.
+
+If you no longer have this command, or the token has expired, you can run `kubeadm token create --print-join-command` (on a control plane host) to generate a new token and join command.
+```
+Example:
+kubeadm join 192.168.101.5:6443 --token tdp0jt.rshv3uobkuoobb4v     --discovery-token-ca-cert-hash sha256:84a163e57bf470f18565e44eaa2a657bed4da9748b441e9643ac856a274a30b9
+```
+
+#### Verify your installtion
+There will be temporary network interruption on windows worker node on the first startup of Antrea Agent. It's because Antrea Agent will set the OVS to take over the host network. Then you should now be able to view the Windows nodes and Pods in your cluster by running:
+
+```
+# Show nodes
+kubectl get nodes -o wide -nkube-system
+NAME                           STATUS   ROLES    AGE    VERSION   INTERNAL-IP     EXTERNAL-IP   OS-IMAGE                                  KERNEL-VERSION     CONTAINER-RUNTIME
+win-5akrf2tpq91                Ready    <none>   1h   v1.17.0   10.176.27.150   <none>        Windows Server 2019 Standard Evaluation   10.0.17763.1158    docker://19.3.5
+win-5akrf2tpq92                Ready    <none>   1h   v1.17.0   10.176.27.197   <none>        Windows Server 2019 Standard Evaluation   10.0.17763.1158     docker://19.3.5
+
+# Show Antrea Agent and kube-proxy pods
+kubectl get pods -o wide -nkube-system | grep windows
+antrea-agent-windows-6hvkw                             1/1     Running     0          100s
+kube-proxy-windows-2d45w                               1/1     Running     0          102s
+```
+
+
+## Troubleshooting
+1. HNS Network is not persistent on Windows. So after the Windows Node is restarted,
+the HNS Network create by Antrea Agent is removed, and the Open vSwitch Extension is
+disabled by default. In this case, the stale OVS bridge and ports should be removed
```

Comment:
can you provide the necessary commands here?

once the "stale" OVS bridge has been removed, what do I need to do? I'm confused by what "running Antrea Agent" means here, wouldn't the Agent start automatically on restart, since "antrea-windows.yml" was applied to the cluster?

Reply:
The description was wrote when before we use the daemon set. Sure, I should update it.

We indeed don't need to `run Antrea Agent` manually. And Antrea Agent should clean the stale OVS on startup. I will draft a patch for it. Then we will don't need to do the clean operations manully.

Add `TODO` here. I will update the description later.